crafters/audio/AudioReader/README.md

```diff
@@ -1,3 +1,53 @@
 # AudioReader
 
-AudioReader reads and resamples the audio signal on doc-level. 
\ No newline at end of file
+**AudioReader** reads and resamples audio signal on doc-level.  Loads an audio file as `ndarray` and resamples the audio signal to the target sampling rate(default 22050Hz).
+
+The supported sound formats are:
+WAV, MP3, OGG, AU,FLAC, RAW, AIFF, AIFF-C, PAF, SVX, NIST, VOC, IRCAM, W64, MAT4, MAT5, PVF, XI, HTK, SDS, AVR, WAVEX, SD2, CAF, WVE, MPC2K, RF64.
+
+## Usage
+
+The **AudioReader** executor needs only one parameter:
+
+| `param_name`  | `param_remarks` |
+| ------------- | ------------- |
+| `target_sample_rate`  | The desired sample rate. Requires a scalar number bigger than 0  |
+
+
+We use Pod images in several ways:
+
+1. Run with Docker: `docker run`
+   ```bash
+    docker run jinahub/pod.crafter.audioreader:0.0.8-1.0.1 --port-in 55555 --port-out 55556
+    ```
+    
+2. Run with Flow API
+   ```python
+    from jina.flow import Flow
+
+    f = (Flow()
+        .add(name='my-encoder', image='jinahub/pod.crafter.audioreader:0.0.8-1.0.1', port_in=55555, port_out=55556))
```

Comment:
```suggestion
        .add(name='my_encoder', uses='docker://jinahub/pod.crafter.audioreader:0.0.8-1.0.1', port_in=55555, port_out=55556))
```

Reply:
We also need "docker" here? it's not specified in our docs, but if we do need it, we also need to change the docs https://docs.jina.ai/chapters/hub/build-pod-image.html

Reply:
This doc is not updated. Same with this [one](https://github.com/jina-ai/docs/blob/ed9472013ddf43370dde4669be5f0f0379053e65/chapters/hub/use-your-pod.md).
the 'image='image_name' is not outdated and we should use "uses='docker://image_name" instead  of that one. And if you run this locally. The first error will be the my-encode name, the second issue is it won't use the image, although there are no errors.

```
f = (Flow()
        .add(name='my-encoder', image='jinahub/pod.crafter.audioreader:0.0.8-1.0.1', port_in=55555, port_out=55556)) 

```